examples/25_system_information.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import cv2
+import depthai as dai
+
+
+def print_sys_info(info):
+    m = 1024 * 1024 # MiB
+    print(f"Drr used / total - {info.ddrMemoryUsage.used / m:.2f} / {info.ddrMemoryUsage.total / m:.2f} MiB")
+    # Add binding first
+    # print(f"Cmx used / total - {info.cmxMemoryUsage.used / m:.2f} / {info.cmxMemoryUsage.total / m:.2f} MiB")
```

Comment:
This can be uncommented now

Reply:
I thought there would be automatic checks that would throw an error